scripts/qa_crowbarsetup.sh

```diff
@@ -3068,6 +3087,12 @@ function get_ceph_nodes
     fi
 }
 
+function oncontroller_get_network_id
+{
+    local network_id=$(openstack network show fixed -f value -c id)
+    echo $network_id
```

Comment:
You could combine those two lines into one, i.e.
```
openstack network show fixed -f value -c id
```

Reply:
fixed